src/control/cmd/daos_server/network.go

```diff
@@ -79,6 +79,10 @@ func GetLocalFabricIfaces(ctx context.Context, fs *hardware.FabricScanner, filte
 }
 
 func (cmd *networkScanCmd) Execute(_ []string) error {
+	if err := common.CheckDupeProcess(); err != nil {
```

Comment:
I'm trying to be consistent in the behaviour of all the `daos_server` commands, when a service is up and running it makes sense to use `dmg network scan` rather than the local variant. I'm happy to change the behaviour if you feel strongly about it though.

Reply:
OK, that's fair. I don't feel strongly, was just curious about whether there was some technical reason for it that I wasn't seeing.